web/src/main/resources/templates/fragments/expressions-of-interest-tables.html

```diff
@@ -170,45 +170,100 @@ <h1 class="table-heading">Accepted Requests To Supply Services</h1>
     <hr/>
 </div>
 
+<!-- Partner can view and apply for Expressions of Interest that are still open. -->
+<div th:fragment="partner-pending-expressions-of-interest" id="pending-expressions-of-interest" xmlns:th="http://www.thymeleaf.org">
+    
+    <div class="row">
+        <div class="col-12 col-md-4 col-lg-4 col-xl-3"><h1 class="table-heading">Open Expressions Of Interest</h1></div>
+        <div class="col-12 col-md-6 col-lg-6 col-xl-6"><input class="form-control" id="pendingEOIFilter" type="text" name="keyword" placeholder="Filter Expressions of Interest..." /> </div>
+        <div class="col-md-1 col-xl-3"></div>
+    </div>
+    
+    <div class="row table-container">
+        <table class="table table-sm table-primary table-striped table-hover">
+            <thead>
+                <tr>
+                    <th>ID</th>
+                    <th>Category</th>
+                    <th>Location</th>
+                    <th>Description</th>
+                    <th>Date Opened</th>
+                    <th class="text-center">Manage</th>
```

Comment:
I don't think educators need access to edit or delete an expression of interest
```suggestion
<th class="text-center"></th>
```